org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/factory/DefaultFeedbackService.java

```diff
@@ -113,10 +115,20 @@ protected void showEmailDialog(byte[] imageBytes) {
 			dialog.setAttribute(Window.MODE_KEY, Mode.OVERLAPPED);			
 			
 			MSystem system = MSystem.get(Env.getCtx());
-			if (!Util.isEmpty(system.getSupportEMail())) 
+
+			for (String s : getFeedbackRecipient(MSysConfig.FEEDBACK_EMAIL_TO).split(","))
+				if(EMail.validate(s.trim()))
+					dialog.addTo(s.trim(), Util.isEmpty(dialog.getTo()));
+
+			if (Util.isEmpty(dialog.getTo()) && !Util.isEmpty(system.getSupportEMail())) 
 			{
 				dialog.addTo(system.getSupportEMail(), true);
 			}
+
+			for (String s : getFeedbackRecipient(MSysConfig.FEEDBACK_EMAIL_CC).split(","))
+				if(EMail.validate(s.trim()))
+					dialog.addCC(s.trim(), Util.isEmpty(dialog.getCc()));
```

Comment:
same here, Util.isempty check is unnecessary, should just pass false as parameter.